packages/datagrid/stories/datagrid.component.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { IconsProvider } from '@talend/react-components';
 
+import './config/i18n';
```

Comment:
Why you need to import this now? Don't understand, stories don't seem to be broke on datagrid

Reply:
I think it's needed to make the `i18next.init()` code run, it's not exported but just run in the global scope so it's enough to do an import. Correct me if I'm wrong